android/titanium/src/java/org/appcelerator/titanium/TiBaseActivity.java

```diff
@@ -631,6 +631,7 @@ public void setContentView(View view, LayoutParams params)
 	@Override
 	protected void attachBaseContext(Context newBase)
 	{
+		baseContext = newBase;
```

Comment:
Shouldn't this be released / cleaned up again at some point?

Reply:
:thinking: I think only when the app is closed/killed but then it should be gone anyways. During the runtime it should be there all the time since its the root/source activity.

Reply:
But since there is a `onDestroy` I'll do a quick test if it still works when I put it into that and clear it

Reply:
fast builds now :smile: 

Added 
```
if (baseContext != null) {
	baseContext = null;
}
```
to `onDestroy()`. It's going into it when closing the app. Restarting it still works fine with the print module. Feels better like this. Added a second window to my printer test app to and that still works correct too. 

Tested: 
* win1 with all the content, opening win2, closing win2, calling printer in win1
* win2 with all the content, opening win2, calling printer in win2, closing win2